docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -60,6 +61,26 @@ function_call ::= function_name '(' [ arguments ... ] ')'
 | MIN | Returns the minimum value of column values |
 | MAX | Returns the maximum value of column values |
 
+
+## Partition_hash function
+YugabyteDB uses `partition_hash` function that takes as arguments the values of the `partition keys` and returns an `uint16` 
```

Comment:
No -- this description is wrong. We'll need to get someone else to write it.

Reply:
See update on latest commit

Reply:
@kmuthukk reminder for review